packages/extension-api/src/extension-api.d.ts

```diff
@@ -827,7 +827,7 @@ declare module '@podman-desktop/api' {
     /**
      * A description of the field to be show (Markdown format)
      */
-    markdownDescription?;
+    markdownDescription?: string;
```

Comment:
maybe issue a small PR with the changes related to adding the missing types as it should work out of the box

this one, new NotificationType, IPAM interface, DeviceRequest etc

Reply:
Yes, this PR was suppose to be small enough for those changes, but it start to get too big. Agreed